### PR TITLE
Add changes to make UPF-EPC work in K8S environment

### PR DESCRIPTION
**Pull request by hyunsun**

### CODE REVIEW

File: conf/spgwu.bess [hunk 1]
```diff
@@ -50,6 +50,8 @@ def scan_dpdk_ports():
             else:
                 raise
         idx += 1
+        if idx == 30:
```
Comment (krsna1729) [inline conf/spgwu.bess]:
```suggestion
        # RTE_MAX_ETHPORTS is 32 and we need 2 for vdevs
        if idx == 30:
```

File: conf/route_control.py [hunk 2]
```diff
@@ -305,6 +305,12 @@ def probe_addr(item, src_mac):
 
 def parse_new_route(msg):
     item = NeighborEntry()
+    # Fetch prefix_len
+    item.prefix_len = msg['dst_len']
```
Comment (krsna1729) [inline conf/route_control.py]: These two lines are duplicated. Need to remove the occurrence around 338 339

File: pod.yaml [hunk 3]
```diff
@@ -37,9 +55,12 @@ spec:
         exec:
           command: ["sh", "-c", "until bessctl run spgwu; do sleep 2; done;"]
     securityContext:
+      # TODO: fix DPDK port detection issue and remove priviledged
+      privileged: true
       capabilities:
         add:
-          - IPC_LOCK # AF_PACKET vdev uses mmap
+        - NET_ADMIN
```
Comment (krsna1729) [inline pod.yaml]:
```suggestion
        - NET_ADMIN # To add routes post-startup
```

File: conf/spgwu.bess (hunk 1 above)
Comment (ajamshed) [inline conf/spgwu.bess]: Any reason why the `idx` check is hard-coded till 30?

Comment (krsna1729) [reply conf/spgwu.bess]: @hyunsun somehow the accepted suggestion got disappeared.
@ajamshed 
```
 # RTE_MAX_ETHPORTS is 32 and we need 2 for vdevs
```

Comment (hyunsun) [reply conf/spgwu.bess]: Hmm.....I'll include it to my patch.

**Pull request merged**